operator/molecule/test-local/asserts.yml

```diff
@@ -0,0 +1,32 @@
+- name: Verify
+  hosts: localhost
+  connection: local
+  vars:
+    ansible_python_interpreter: '{{ ansible_playbook_python }}'
+    deploy_dir: "{{ lookup('env', 'MOLECULE_PROJECT_DIRECTORY' }}/deploy"
+    custom_resource: "{{ lookup('file', '/'.join([deploy_dir, 'crds/osb_v1alpha1_automationbroker_cr.yaml']) || from_yaml }}"
+  tasks:
+    - block:
```

Comment:
This block isn't necessary, unless you're planning on adding a rescue that prints debugging information or something